AI-and-Analytics/Getting-Started-Samples/LPOT-Sample-for-Tensorflow/lpot_sample_tensorflow.ipynb

```diff
@@ -354,7 +363,7 @@
    },
    "outputs": [],
    "source": [
-    "!python lpot_quantize_model.py"
+    "!TF_ENABLE_MKL_NATIVE_FORMAT=0 python lpot_quantize_model.py"
```

Comment:
Hi
Do we need TF_ENABLE_MKL_NATIVE_FORMAT=0 if we have export it via the cell before this one?
maybe we only need to have either one of them.

Reply:
Yes.
But due to this variable is mandatory to LPOT now, I want to high light it in the cmd.
Sometime, user will copy the cmd line in new term to run it. Maybe the current cmd is easy to copy/paste and avoid to miss the variable setting.